src/components/DashboardLayout.tsx

```diff
@@ -34,16 +39,18 @@ const Layout: FunctionComponent<Props> = (props, { children }) => {
           <strong className={`lbh-heading-h5 ${styles['switcher__name']}`}>
             {currentTeam?.name}
           </strong>
-          <Link
-            href={{
-              pathname: '/teams',
-              query: { teamId: props.teamId },
-            }}
-          >
-            <a className={`lbh-link ${styles['switcher__link']}`}>
-              Switch team
-            </a>
-          </Link>
+          {userTeams.length !== 1 && (
```

Comment:
You could make this greater than 1 so it's a bit more intuitive when someone reads the condition
```suggestion
          {userTeams.length > 1 && (
```